clang/lib/DPCT/Diagnostics.inc

```diff
@@ -204,8 +204,8 @@ DEF_WARNING(SUBGROUP_SIZE_CONFLICT, 1085, LOW_LEVEL, "The function %0 requires s
 DEF_COMMENT(SUBGROUP_SIZE_CONFLICT, 1085, LOW_LEVEL, "The function {0} requires sub-group size to be {1}, while other sub-group functions in the same SYCL kernel require a different sub-group size. You may need to adjust the code.")
 DEF_WARNING(ACTIVE_MASK, 1086, HIGH_LEVEL, "__activemask() is migrated to 0xffffffff. You may need to adjust the code.")
 DEF_COMMENT(ACTIVE_MASK, 1086, HIGH_LEVEL, "__activemask() is migrated to 0xffffffff. You may need to adjust the code.")
-DEF_WARNING(ND_RANGE_BARRIER, 1087, LOW_LEVEL, "SYCL currently does not support cross group synchronization. You can specify \"--use-experimental-features=nd_range_barrier\" to use the dpct helper function nd_range_barrier to migrate %0.")
-DEF_COMMENT(ND_RANGE_BARRIER, 1087, LOW_LEVEL, "SYCL currently does not support cross group synchronization. You can specify \"--use-experimental-features=nd_range_barrier\" to use the dpct helper function nd_range_barrier to migrate {0}.")
+DEF_WARNING(ND_RANGE_BARRIER, 1087, LOW_LEVEL, "The nd_range_barrier is deprecated, please try to remigrate with option: --use-experimental-features=root-group")
```

Comment:
the def of 1087 is totally changed.  Let's update the 1087 to use "root-group feature"  only.

Reply:
SYCL currently does not support cross group synchronization. You can specify \"--use-experimental-features=root-group\" to use the root-group to migrate {0}